solidity/test/tip-bot.test.js

```diff
@@ -0,0 +1,98 @@
+const truffleAssertions = require("truffle-assertions");
+const TipBot = artifacts.require("TipBot");
+
+contract("TipBot", (accounts)=>{
+    const [ admin1, admin2, user1, user2, user3, user4 ] = accounts;
+    before( async ()=>{
+        tipbot = await TipBot.deployed();
+        tipbot.grantRole(web3.utils.fromAscii("DEFAULT_ADMIN_ROLE"), admin1);
+        tipbot.grantRole(web3.utils.fromAscii("DEFAULT_ADMIN_ROLE"), admin2);
+    }); 
+
+    it("Should have a feeRate of 0.1 eth", async () => {
+        const feeRate = await tipbot.feeRate();
+        const feeRateEth = await web3.utils.fromWei(feeRate);
+        console.log(feeRateEth);
+        expect(feeRateEth).to.be.equal('0.1');
+    });
+
+    it("Should have an airDropRate of 0.2 eth", async () => {
+        const feeRate = await tipbot.airdropRate();
+        const feeRateEth = await web3.utils.fromWei(feeRate);
+        console.log(feeRateEth);
+        expect(feeRateEth).to.be.equal('0.2');
+    });
+
+    it("Should be able change fee rate not greater than 1 eth", async() => {
+        await tipbot.setFeeRate( web3.utils.toWei('0.2'));
+        const feeRate = await tipbot.feeRate();
+        const feeRateEth = await web3.utils.fromWei(feeRate);
+        expect(feeRateEth).to.be.equal('0.2');
+    });
+
+    it("Should be able change airDrop rate not greater than 1 eth", async() => {
+        await tipbot.setAirdropRate( web3.utils.toWei('0.4'));
+        const feeRate = await tipbot.airdropRate();
+        const feeRateEth = await web3.utils.fromWei(feeRate);
+        expect(feeRateEth).to.be.equal('0.4');
+    });
+
+    it("Should be able to tip user1 to user2", async () => {
+
+        var user1_initial_balance = await web3.eth.getBalance(user1);
```

Comment:
use `const` for one-item variable assignments otherwise use`let`

Reply:
Modified using `let`